frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -240,6 +240,7 @@ class PivotTable extends Component {
       title: t`Sort Order`,
       widget: ChartSettingIconRadio,
       inline: true,
+      borderBottom: true,
```

Comment:
The design [is discussed here](https://metaboat.slack.com/archives/C03SD8HDJLC/p1669255147605829).